packages/monitoring/src/index.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+import express, { Request, Response } from 'express';
+
+import client from 'prom-client';
+
+const prometheusApp = express();
+
+
+/**
+ * A registry to manage metrics
+ */
+const register = new client.Registry();
+
+/**
+ * Collect the default metrics that prometheus gives
+ */
+client.collectDefaultMetrics({
+  labels: { app: 'node-application-monitoring-app' },
+  prefix: 'node_',
+  register
+});
+
+/**
+ * A histogram for request details which is counted based on bucket values.
+ */
+const httpRequestTimer = new client.Histogram({
+  name: 'http_request_duration_seconds',
+  help: 'Duration of HTTP requests in seconds',
+  labelNames: [ 'method', 'route', 'code' ],
+  buckets: [ 0.1, 0.3, 0.5, 0.7, 1, 3, 5, 7, 10 ]
+});
+
+/**
+ * Register the histogram into the registry, The registry provides a set of APIs that can be used to interact with histogram metrics.
+*/
+register.registerMetric(httpRequestTimer);
+
+
+/**
+ * A function to initialize the prometheus service. It will monitor the request duration by storing route, status code and request method of each request.
+ * @param PROMETHEUS_PORT Port which prometheus will listen
+ * @param routeToExpose Route on which server will expose the metrics
+ * @returns void
+ */
+export function initPrometheus(PROMETHEUS_PORT: number, routeToExpose: string) {
```

Comment:
as per the example provided above in readme, second param should be optional either or should update in readme too.